### ISSUE TITLE
Improve language selector filtering

### ISSUE THREAD

**Issue by ashklianko** [enonic/app-contentstudio#4210]
@sigdestad commented on [Wed Jan 12 2022](https://github.com/enonic/lib-admin-ui/issues/2366)

Currently, when typing in locales, you do not get the expected results. Try typing "no" for instance, and you get the following:
![Screenshot 2022-01-12 at 10 00 06](https://user-images.githubusercontent.com/482228/149096668-d59d5294-3369-49ca-ae5e-be73cf89a646.png)

Change the filter, so that locales get top-priority when filtering the results.

I.e. search for "no" should give a list like this:

Norsk (no)
Nynorsk (no-ny)
...
Italiano (it)

Basically, sort by locale first, then label
